__tests__/PetClass.test.js

```diff
@@ -0,0 +1,226 @@
+const { Pet, petConfig } = require('../src/PetClass');
+
+const {
+  AGE_INIT,
+  AGE_INCREMENT,
+  AGE_MAX,
+  FITNESS_INIT,
+  FITNESS_MIN,
+  FITNESS_INCREMENT,
+  FITNESS_DECREMENT,
+  FITNESS_THRESHOLD,
+  HUNGER_INIT,
+  HUNGER_MAX,
+  HUNGER_THRESHOLD,
+  HUNGER_INCREMENT,
+  HUNGER_DECREMENT
+} = petConfig;
+
+describe('Pet class', () => {
+  it('returns an object', () => {
+    expect(new Pet()).toBeInstanceOf(Object);
+  });
+
+  it('returns an instance of Pet', () => {
+    expect(new Pet()).toBeInstanceOf(Pet);
+  });
+
+  it('throws TypeError if the name argument is not <string>', () => {
+    expect(() => new Pet(42)).toThrow(TypeError);
+    expect(() => new Pet(false)).toThrow('name must be a string or undefined');
+  });
+});
+
+describe('Pet instance', () => {
+  it('name initialised as name argument (if provided)', () => {
+    expect(new Pet('Ollie').name).toBe('Ollie');
```

Comment:
While this short format works, generally its nice to separate out the thing you are testing from the assertion block so its easier to see at a glance what it is you are creating as well as what you are testing from that. In this case it works fine, just. something to be aware of when it comes to testing more complicated things in the future is all